drivers/gles2/shader_compiler_gles2.cpp

```diff
@@ -619,6 +619,19 @@ String ShaderCompilerGLES2::_dump_node_code(SL::Node *p_node, int p_level, Gener
 		case SL::Node::TYPE_OPERATOR: {
 			SL::OperatorNode *op_node = (SL::OperatorNode *)p_node;
 
+			if (op_node->op == SL::OP_ASSIGN) {
```

Comment:
It was fine before, but you had to mark the fallthrough as intentional with `FALLTHROUGH;`.

Reply:
Ah, right... forgot about this thing.